src/docs/wtperf.dox

```diff
@@ -128,6 +128,8 @@ configuration options:
 DO NOT EDIT: THIS PART OF THE FILE IS GENERATED BY dist/s_docs.
 \endif
 
+@par backup_complete (unsigned int, default=0)
+when doing backups, either just perform source-side reading or take a complete backup
 @par backup_interval (unsigned int, default=0)
```

Comment:
This is unrelated to my change. It came from running `s_docs` (via `s_all`).

Reply:
That's from my recent merge. But `s_docs` or `s_all` did not fail on my PR.